lib/Firtool/Firtool.cpp

```diff
@@ -386,6 +386,7 @@ LogicalResult firtool::populateFinalizeIR(mlir::PassManager &pm,
 LogicalResult firtool::populateHWToBTOR2(mlir::PassManager &pm,
                                          const FirtoolOptions &opt,
                                          llvm::raw_ostream &os) {
+  pm.addNestedPass<hw::HWModuleOp>(circt::createLowerLTLToCorePass());
```

Comment:
Register this pass over in `tools/firtool/firtool.cpp` next to the `registerConvertHWToBTOR2Pass();` line